autopush/router/fcm.py

```diff
@@ -72,11 +151,17 @@ def route_notification(self, notification, uaid_data):
     def _route(self, notification, router_data):
         """Blocking FCM call to route the notification"""
         data = {"chid": notification.channel_id}
+        if not router_data.get("token"):
+            raise self._error("No registration token found. "
+                              "Rejecting message.",
+                              410, errno=106, log_exception=False)
+        self.regid = router_data.get("token")
         # Payload data is optional. The endpoint handler validates that the
         # correct encryption headers are included with the data.
         if notification.data:
             mdata = self.config.get('max_data', 4096)
-            if len(notification.data) > mdata:
+            self.notifLen = len(notification.data)
```

Comment:
Crap. yeah, leftover from a previous thought. I can revert this back and drop notifLen entirely, since I'm passing notification to _process_reply()